src/NosCore.Handler/CommandPacketHandler.cs

```diff
@@ -29,13 +33,45 @@ public CommandPacketHandler(ClientSession session)
         #region Methods
         public void Speed(SpeedPacket speedPacket)
         {
-            if (speedPacket.Speed >= 60)
+            if (speedPacket.Speed > 0 || speedPacket.Speed < 60)
             {
-                return;
+                if (speedPacket.Speed >= 60)
+                {
+                    return;
+                }
+                Session.Character.Speed = speedPacket.Speed;
+                Session.SendPacket(Session.Character.GenerateCond());
+            }
+            else
+            {
+                Session.SendPacket(Session.Character.GenerateSay(speedPacket.Help(), 10));
+            }
+        }
+
```

Comment:
In the Help command, maybe adding an authority check would be a good idea (at least I don't see any), to only display available commands to the user, as an example, a GameMaster doesn't need to see Administrator commands and so on

Reply:
This is already in the packet itself

Reply:
Oh actually no I didn’t understood what you mean could be usefull yes